packages/lexical-editor/src/components/ToolbarActions/FontSizeAction.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { useLexicalComposerContext } from "@lexical/react/LexicalComposerContext";
+import { $getSelectionStyleValueForProperty, $patchStyleText } from "@lexical/selection";
+import { mergeRegister } from "@lexical/utils";
+import {
+    $getSelection,
+    $isRangeSelection,
+    COMMAND_PRIORITY_CRITICAL,
+    LexicalEditor,
+    SELECTION_CHANGE_COMMAND
+} from "lexical";
+import { DropDown, DropDownItem } from "../../ui/DropDown";
+
+/**
+ * Toolbar action. Allow user to change font size for selected text.
+ */
+
+const FONT_SIZE_OPTIONS: [string, string][] = [
+    ["8px", "8px"],
+    ["9px", "9px"],
+    ["10px", "10px"],
+    ["11px", "11px"],
+    ["12px", "12px"],
+    ["14px", "14px"],
+    ["15px", "15px"],
+    ["16px", "16px"],
+    ["18px", "18px"],
+    ["21px", "21px"],
+    ["24px", "24px"],
+    ["30px", "30px"],
+    ["36px", "36px"],
+    ["48px", "48px"],
+    ["60px", "60px"],
+    ["72px", "72px"],
+    ["96px", "96px"]
+];
```

Comment:
Is there a reason why this is structured as tuples, and not just array of strings? I mean, there doesn't ever seem to be a different between the "label" and "value".

Reply:
yup, works with regular array. The reason is that in a original code the toolbar actions are not separated in different components so they adjusted the data to fit all actions that have this kind of logic.